pkg/runtimeproxy/server/docker/handler.go

```diff
@@ -114,7 +118,7 @@ func (d *RuntimeManagerDockerServer) HandleCreateContainer(ctx context.Context,
 		NetworkingConfig: networkingConfig,
 	}
 
-	if runtimeResourceType == resource_executor.RuntimePodResource && hookResp != nil {
```

Comment:
Check `hookResp is nil` is very important. I found that remove it could lead to type conversion got nil exception.